utils/MyWalker.py

```diff
@@ -24,7 +24,7 @@ def queue_children(self, element: Container) -> None:
             return
 
         element = element.reload()
-        log.info(
+        log.debug(
```

Comment:
Are you using a custom walker to reload each container?

Reply:
Yes...I think when I made this the walker didn't reload on it's own.  has that changed?